ingested_data/meta/proposal_builder.py

```
@@ -63,7 +63,7 @@ def load_spec_content(spec_paths):
 def md_section(title: str) -> str:
     return f"\n\n## {title}\n\n"
 
-def build_proposal(snap: dict, forced: bool) -> str:
+def build_proposal(snap: dict, forced: bool, spec_data: dict = None) -> str:
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)